apps/chat/src/components/Chat/Migration/Migration.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { useTranslation } from 'next-i18next';
+
+import { Translation } from '@/src/types/translation';
+
+import { Spinner } from '@/src/components/Common/Spinner';
+
+interface Props {
+  total: number;
+  uploaded: number;
+}
+
+export const Migration = ({ total, uploaded }: Props) => {
+  const { t } = useTranslation(Translation.Common);
+
+  return (
+    <div className="flex h-full flex-col items-center justify-center">
+      <Spinner className="h-auto" size={60} />
+      <h1 className="mt-7 text-2xl font-semibold md:text-3xl">
+        {uploaded} {t('out of')} {total}
```

Comment:
Your heading should be fully meaningful otherwise you will have bad SEO and other aspects

Add heading Migration as h1 and other info as p